cmake/codecheck/rules/format_TODO_comments.py

```diff
@@ -85,7 +85,7 @@ def evaluate_matches(lines, fn):
     '* TODO: This is a todo comment',
     '/// TODO: This is a todo comment',
     '//! TODO: This is a todo comment',
-    '\TODO: This is a todo comment',
+    r'\TODO: This is a todo comment',
     '\\\todo: This is a todo comment'
```

Comment:
You missed just this one:
```suggestion
   r'\\\todo: This is a todo comment'
```

Reply:
Thanks :-)